packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -157,6 +176,7 @@ const orderFormToCart = async (
       text,
       status: status.toUpperCase(),
     })),
+    totalizers: form.totalizers,
```

Comment:
Why do we need this? That's not clear to me. Also, have we tried to fit it into a more agnostic schema? It seems to me that `totalizers` is a VTEX abstraction.

Reply:
Removed! My mistake 🧼